doc/source/ray-core/fault_tolerance/ray_components.rst

```diff
@@ -0,0 +1,69 @@
+.. _fault-tolerance-internal-system:
+
+Advanced topic: Ray system failure model
```

Comment:
@stephanie-wang I think this page could use a general editing pass.

Reply:
Discussed with @stephanie-wang offline, this page is removed from this PR and will be added in a follow-up.